app/helpers/navigation_helper.rb

```diff
@@ -0,0 +1,18 @@
+module NavigationHelper
+  def navigation_links_internal
```

Comment:
They're separate methods because they are used by both the legacy layout and the design system layout.  In the legacy layout, the links are in separate places in the UI:

<img width="1161" alt="image" src="https://github.com/alphagov/manuals-publisher/assets/138604938/fc571035-f8c5-4b69-bb9a-15fcc38aa877">

Reply:
Oh sure, sorry I hadn't clocked that, a few of my comments won't have made sense then.